### PR TITLE
Feature/remove release note

### PR DESCRIPTION
**Pull request by Mordi490**
Removing a release note returns the info of the deleted release note

### CODE REVIEW

File: Controllers/ReleaseNoteController.cs [hunk 1]
```diff
@@ -67,5 +67,27 @@ public async Task<ActionResult<ReleaseNoteResource>> UpdateReleaseNoteAsync(int
...
+        [HttpDelete("{id}")]
+        [Authorize(Roles = ("Administrator"))]
+        public async Task<ActionResult<ReleaseNoteResource>> RemoveReleaseNote(int id)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState.GetErrorMessages());
+            }
+            
+            //var existingReleaseNote = GetSpecificReleaseNote(id);
```
Comment (Mordi490) [inline Controllers/ReleaseNoteController.cs]: oops

File: TestContainer/test/testReleases.js [hunk 2]
```diff
@@ -150,4 +151,39 @@ describe("Releases GET", () => {
         done();
       });
   });
+
+  // failure case for getting a single release due to not logging in, 401 unauth
+  it("GET | Should return a 401 unauth", done => {
+    chai
+      //console
+      //.log(["DATA FROM A SINGLE RELEASE1: ", res.text])
```
Comment (markusranda) [inline TestContainer/test/testReleases.js]: Den treng du vel ikke

**Review by markusranda — CHANGES_REQUESTED** [review]
Såg bra ut.
Rydd bort skrotet.

File: TestContainer/test/testReleases.js (hunk 2 above)
Comment (Mordi490) [reply TestContainer/test/testReleases.js]: True

**Review by 0us — APPROVED** [review]

**Pull request merged**